src/components/DrawerMenu.vue

```diff
@@ -21,18 +21,23 @@
             </v-list-item-content>
           </v-list-item>
         </template>
+        <v-list-item> <create-a-list></create-a-list></v-list-item>
       </v-list>
     </v-navigation-drawer>
   </div>
 </template>
 
 <script>
+import CreateAListVue from './CreateAList.vue';
+
 export default {
   name: 'DrawerMenu',
   props: ['drawer'],
+  components: {
+    'create-a-list': CreateAListVue,
```

Comment:
The satandard naming convention is to use the camel-case version of the name of the custom element. 
```javascript
components: {
  CreateAList
}
```
The name of the element should be in the name property of the component, so it is unnecessary here. Less code means efwer bugs.